tests/report/enc/enc.cpp

```diff
@@ -43,12 +47,40 @@ void test_minimum_issue_date(oe_datetime_t now)
             0,
             NULL,
             0,
-            report,
+            &report,
             &report_size) == OE_OK);
 
     // Verify the report.
     OE_TEST(oe_verify_report(report, report_size, NULL) == OE_OK);
 
+    // Generate reports.
+    OE_TEST(
+        oe_get_report_v1(
+            OE_REPORT_FLAGS_REMOTE_ATTESTATION,
+            NULL,
+            0,
+            NULL,
+            0,
+            report_v1,
+            &report_v1_size) == OE_OK);
+
+    // Verify the report.
+    OE_TEST(oe_verify_report(report_v1, report_v1_size, NULL) == OE_OK);
+
+    // Generate reports.
+    OE_TEST(
+        oe_get_report_v2(
+            OE_REPORT_FLAGS_REMOTE_ATTESTATION,
+            NULL,
+            0,
+            NULL,
+            0,
+            &report_v2,
+            &report_v2_size) == OE_OK);
+
+    // Verify the report.
+    OE_TEST(oe_verify_report(report, report_size, NULL) == OE_OK);
```

Comment:
verify report_v2 instead?

Reply:
there is no _v2 version of verify as it does not return a buffer. Or does it? Dave did not mention it in the bug.

Reply:
I mean the variable `report_v2` since you call get report on `report_v2` but then you verify the `report` variable instead.